### PR TITLE
refactor: Make flobject consistent with server.

### PR DESCRIPTION
**Pull request by prmukherj**
This PR can be merged once server-side changes are in place.

### CODE REVIEW

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -33,10 +33,10 @@
 
 
 class InactiveObjectError(RuntimeError):
-    """Provides the error when the object is inactive."""
+    """Raised on attempt to access an inactive object."""
```
Comment (hpohekar) [inline src/ansys/fluent/core/solver/flobject.py:36]: Class descriptions begin with verbs ending with "s" or "es".

Comment (prmukherj) [reply src/ansys/fluent/core/solver/flobject.py:36]: Then something like gets raised? Doesn't sound right.
cc. @seanpearsonuk

Comment (prmukherj) [reply src/ansys/fluent/core/solver/flobject.py:36]: What about: "Provides an error message on attempt to access an inactive object" ?

Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:36]: Provides the error when an inactive object is accessed.

Comment (seanpearsonuk) [reply src/ansys/fluent/core/solver/flobject.py:36]: @prmukherj either what you have written or
"Inactive object access"
```
>>> help(ValueError)
Help on class ValueError in module builtins:

class ValueError(Exception)
 |  Inappropriate argument value (of correct type).
...

>>> help(RuntimeError)
Help on class RuntimeError in module builtins:

class RuntimeError(Exception)
 |  Unspecified run-time error.
...

class TypeError(Exception)
 |  Inappropriate argument type.
```
etc

I don't think we actually want to say "Provides the error when..." as we have in the current code.

**Review by hpohekar — APPROVED** [review]

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -33,10 +33,10 @@
 
 
 class InactiveObjectError(RuntimeError):
-    """Provides the error when the object is inactive."""
+    """Inactive object access.."""
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:36]: Looks like a bonus .
```suggestion
    """Inactive object access."""
```

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**